File: belajarUTS/cleanDuck.py
print("Welcome to CleanDUCK Indonesia!")

print("\nSilahkan Masukkan Data Pelanggan:")
nama = input("Nama: ")

valid = False
while not valid:
    try:
        no_tlp = int(input("Nomor Telepon: "))  # Menggunakan input() dan mengkonversi ke int
        valid = True  # Input valid, keluar dari loop
    except ValueError:
        print("Input salah. Harap masukkan nomor telepon berupa angka.")

tim_bola = input("Tim Bola Kesayangan: ")

print(f"\nHalo {nama}, silahkan pilih layanan jasa yang Anda inginkan:")
print("1. Premium Washing Rp 100.000")
print("2. Washing and Wax Rp 120.000")
print("3. Full Package Rp 550.000")
pilihan = int(input("Pilihan anda: "))  # Menggunakan input() dan mengkonversi ke int

layanan = ""
if pilihan == 1:
    layanan = "1. Premium Washing, Rp 100.000,00"
elif pilihan == 2:
    layanan = "2. Washing and Wax, Rp 120.000,00"
elif pilihan == 3:
    layanan = "3. Full Package, Rp 550.000,00"
else:
    layanan = "Pilihan tidak valid"  # Menangani pilihan yang tidak valid

print(f"\nPelanggan: {nama}")
print(f"Nomor Telepon: {no_tlp}")
print(f"Tim Bola Kesayangan: {tim_bola}")
print(f"Layanan: {layanan}")

File: belajarUTS/cleanDuck.java
package belajarUTS;

import java.util.Scanner;

public class cleanDuck {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Welcome to CleanDUCK Indonesia!");

        System.out.println("\nSilahkan Masukkan Data Pelanggan:");
        System.out.print("Nama : ");
        String nama = sc.nextLine();

        String noTelp;
        while (true) {
            System.out.print("No Telp: ");
            noTelp = sc.nextLine();
            if (noTelp.matches("\\d+")) {
                break;
            } else {
                System.out.println("Nomor telepon harus berupa angka. Silahkan coba lagi.");
            }
        }


        System.out.print("Tim Bola Kesayangan: ");
        String timBola = sc.nextLine();

        System.out.println("\nHalo " + nama + ", silahkan pilih layanan jasa\nyang Anda inginkan :");
        System.out.println("1. Premium Washing Rp 100.000");
        System.out.println("2. Washing and Wax Rp 120.000");
        System.out.println("3. Full Package Rp 550.000");
        System.out.print("Pilihan anda: ");
        int pilihan = sc.nextInt();

        String layanan="";
        switch (pilihan) {
            case 1:
                layanan = "1. Premium Washing, Rp 100.000,00";
                break;
            case 2:
                layanan = "2. Washing and Wax, Rp 120.000,00";
                break;
            case 3:
                layanan = "3. Full Package, Rp 550.000,00";
                break;
        }

        System.out.println("\nPelanggan:" + nama);
        System.out.println("Nomor Telepon:" + noTelp);
        System.out.println("Kode:" + timBola);
        System.out.println("Layanan: " + layanan);
    }
}
